### PR TITLE
Add support for media translation of block widgets

### PR DESCRIPTION
**Pull request by strategio**
The tests are failing because it requires a small adjustment in `makeMock` (https://git.onthegosystems.com/wpml-packages/core-api/-/merge_requests/77).

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-8670

### CODE REVIEW

**Comment by strategio** [comment]
@OnTheGoSystems/wpml

File: src/Widgets/Block/DisplayTranslation.php [hunk 1]
```diff
@@ -19,6 +19,12 @@ public function add_hooks() {
 		     ->then( spreadArgs( function ( $content ) use ( $getStringsFromMOFile ) {
 			     $strings = $getStringsFromMOFile( \get_locale() );
 
+			     $imageTranslator = make( \WPML_Media_Translated_Images_Update::class );
```
Comment (brucepearson) [inline src/Widgets/Block/DisplayTranslation.php:22]: @strategio I think this should be moved to Media plugin and Media plugin should hook `widget_block_content` before this.

Comment (gordonsh24) [reply src/Widgets/Block/DisplayTranslation.php:22]: @brucepearson We discussed it with Pierre before when we were working on the first Gutenberg task. I also initially put my code into ST, but Pierre explained that it is better to keep it here. 

1. Now, the whole Gutenberg related code is stored in one place
2. if something changes just in Gutenberg, the compatibility is able to release a fix without WPML

**Review by gordonsh24 — APPROVED** [review]

File: src/Widgets/Block/DisplayTranslation.php (hunk 1 above)
Comment (brucepearson) [reply src/Widgets/Block/DisplayTranslation.php:22]: ok

**Review by brucepearson — APPROVED** [review]

File: src/Widgets/Block/DisplayTranslation.php (hunk 1 above)
Comment (strategio) [reply src/Widgets/Block/DisplayTranslation.php:22]: I have been hesitating, both ways have pros and cons...

In the end, I think it's more correct to move the media filtering to WPML Media.

**Comment by strategio** [comment]
Closing here, I moved the code to https://git.onthegosystems.com/wpml/wpml-media-translation/-/merge_requests/350.

**Pull request closed without merging**